app/main.py

```diff
@@ -1 +1,25 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+    checked_friend = True
+
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except (NotVaccinatedError, OutdatedVaccineError):
+            checked_friend = False
+            return "All friends should be vaccinated"
```

Comment:
If at least one of the friend has problems with vaccines the function should return a message "All friends should be vaccinated".`
So, you alredy have there return 
You don't need additionaly check `checked_friend`